frontend/src/app/(routes)/transfers/components/Summary.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from 'next/image';
+import React from 'react';
+import useGetAssetsAmount from '@/custom-hooks/useGetAssetsAmount';
+import { capitalizeFirstLetter, formatDollarAmount } from '@/utils/util';
+import { useAppSelector } from '@/custom-hooks/StateHooks';
+
+const Summary = ({ chainIDs }: { chainIDs: string[] }) => {
+  const [, available] = useGetAssetsAmount(chainIDs);
+  const nameToChainIDs = useAppSelector((state) => state.wallet.nameToChainIDs);
+  let chainName = 'All Networks';
+  let imageURL = '/all-networks-icon.png';
+  let firstChainName = '';
+
+  Object.keys(nameToChainIDs).forEach((name) => {
+    if (nameToChainIDs[name] === chainIDs[0]) firstChainName = name;
+  });
+  const chainImageURL = useAppSelector(
+    (state) => state.wallet.networks[chainIDs[0]]?.network?.logos?.menu || ''
+  );
+
+  if (chainIDs.length === 1) {
+    chainName = firstChainName;
+    imageURL = chainImageURL;
+  }
+
+  return (
+    <div className="coloured-container relative h-[72px]">
+      <Image
+        src="/printed-color.png"
+        width={560}
+        height={72}
+        alt=""
+        className="absolute left-[86px] top-[-2px]"
+      />
+      <div className="flex w-full justify-between">
+        <div className="flex items-center gap-2">
+          <Image src={imageURL} width={40} height={40} alt={chainName} />
+          <div className="text-sm not-italic font-normal leading-[normal]">
+            {capitalizeFirstLetter(chainName)}
+          </div>
+        </div>
+        <div className="flex items-center gap-2">
+          <div className="font-bold text-2xl">
+            {formatDollarAmount(available)}
+          </div>
+          <div className="text-right text-xs not-italic font-normal leading-[normal]">
+            Total Balance
+          </div>
+        </div>
```

Comment:
Let's remove this section. It's redundant since the total available balance is already displayed in the sidebar. Removing it will improve user experience on smaller screens by eliminating the need to scroll to access the `Send` action button. I am attaching screenshot from smaller screen.
![transfers](https://github.com/vitwit/resolute/assets/72057206/00f82c18-55a6-4b1f-8e97-c73b200a88cc)

Reply:
May be we can make it responsive depending on screen size?

Reply:
About this, can we check with Aishwarya once? She was keen on that design..

Reply:
Removed the redundant data, but left the component as it is it might be useful (if the selected Asset is not in top 4 assets, it's not displayed unless one opens the popup. May we can use that section to show selected Asset data or some animation when tx is happening...)